### PR TITLE
Rent model

### PR DESCRIPTION
**Pull request by ivoguzman23**
## Summary

Add test for rent and user models

I made the rspec tests and I use the factories for it.

## TRELLO Card

https://trello.com/c/tE1n77g0/42-crear-tests-para-el-modelo-de-rent-y-user

### CODE REVIEW

File: spec/models/book_test_spec.rb [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  end
+
+  subject(:book) do
+    Book.new(
+        genre: genre, author: author, image: image, title: title,
+        publisher: publisher, year: year
+    )
+  end
+
+  let(:genre)            { Faker::Lorem.word}
```
Comment (lucasenriquez) [inline spec/models/book_test_spec.rb:30]: You can use factories here

File: spec/factories/users.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :user do
+    first_name            { Faker::Name.first_name }
+    last_name             { Faker::Name.last_name }
+    email                 { Faker::Internet.email }
+    password              { 'password' }
+    password_confirmation { 'password' }
+  end
+end
```
Comment (lucasenriquez) [inline spec/factories/users.rb]: Add a newline at the end of the file

File: spec/models/user_test_spec.rb [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    context 'When the password_confirmation is nil' do
+      subject(:user) do
+        build(:user, password_confirmation: nil)
+      end
+      it 'is not valid' do
+        is_expected.to be_invalid
+      end
+    end
+  end
+end
```
Comment (lucasenriquez) [inline spec/models/user_test_spec.rb]: Add a newline at the end of the file

File: spec/support/factory_bot.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+RSpec.configure do |config|
+    config.include FactoryBot::Syntax::Methods
+end
+
```
Comment (lucasenriquez) [inline spec/support/factory_bot.rb]: Add a newline at the end of the file

**Review by lucasenriquez — CHANGES_REQUESTED** [review]

**Review by lucasenriquez — APPROVED** [review]

**Review by GabrielPiccini — APPROVED** [review]

**Pull request merged**